edfi_api_client/edfi_client.py

```diff
@@ -4,12 +4,18 @@
 from functools import wraps
 from requests.auth import HTTPBasicAuth
 from requests.exceptions import HTTPError
-from typing import Callable, Optional
+from typing import Callable, List, Optional
 
 from edfi_api_client import util
 from edfi_api_client.edfi_endpoint import EdFiResource, EdFiDescriptor, EdFiComposite
 from edfi_api_client.edfi_swagger import EdFiSwagger
 
+import logging
```

Comment:
Add official logging, instead of using print statements. There's a world where we could remove `EdFiClient.verbose_log()` altogether and just show `logging.debug` when `verbose=True`.